src/plugins/intel_cpu/src/nodes/deconv.cpp

```diff
@@ -1120,6 +1206,44 @@ bool Deconvolution::canFuseBias() const {
             (externOutShape ? getParentEdges().size() == 3 : getParentEdges().size() == 2));
 }
 
+void Deconvolution::initSupportedPrimitiveDescriptors() {
+    if (!useACL) {
+        Node::initSupportedPrimitiveDescriptors();
+        return;
+    }
+
+    auto& creatorsMap = BlockedDescCreator::getCommonCreators();
+    auto pushDesc = [&](LayoutType format) {
+        NodeConfig config;
+        config.inConfs.resize(getParentEdges().size());
+        config.outConfs.resize(getOriginalOutputsNumber());
+
+        for (size_t i = 0; i < getParentEdges().size(); ++i) {
+            config.inConfs[i].setMemDesc(
+                // ACL expected equal precision
+                creatorsMap.at(format)->createSharedDesc(getOriginalInputPrecisionAtPort(0), getInputShapeAtPort(i)));
+        }
+        config.outConfs[0].setMemDesc(
+                creatorsMap.at(format)->createSharedDesc(getOriginalOutputPrecisionAtPort(0), getOutputShapeAtPort(0)));
```

Comment:
ACL support only equal precisions on inputs and outputs, isn't it? If yes it makes sense to set getOriginalInputPrecisionAtPort(0) to output as well.

Reply:
corrected